WEB-INF/tags/eventForm.tag

```diff
@@ -0,0 +1,39 @@
+      <%-- Form --%>
+      <form action="${pageContext.request.contextPath}/maintenance" method="POST" accept-charset="UTF-8">
+              <input name="utf8" value="?" type="hidden" />
+
+              <%-- Event Name --%>
+              <div class="field">
+                <label for="eventName">Event Name</label><br/>
+                <input type="text" id="eventName" name="eventName" spellcheck="false" class="wide" placeholder="e.g. Site Wide Maintenance" autofocus />
+              </div>
+
+              <%-- Event Summary --%>
+              <div class="field">
+                <label for="eventSummary">Event Summary</label><br/>
+                <input type="text" id="eventSummary" name="eventSummary" spellcheck="false" class="wide" placeholder="e.g. Event Details..." autofocus />
+              </div>
+              <%-- Event Date --%>
+              <div class="field">
+                <label for="eventDate">Event Date</label><br/>
+                <input type="date" id="eventDate" name="eventDate" class="wide" autofocus />
+              </div>
+              <%-- Event Time --%>
+              <div class="field">
+                <label for="eventTime">Event Time</label><br/>
+                <input type="time" id="eventTime" name="eventTime" class="wide" autofocus />
+              </div>
+
+              <hr/>
+
+              <%-- Submit --%>
+              <div class="buttons">
+                <input type="submit" value="Submit" class="button wide">
+              </div>
+
+              <%-- Reset --%>
+                <div class="buttons">
+                  <input type="reset" value="Reset" class="button wide">
+                </div>
```

Comment:
The `buttons` div is for wrapping multiple buttons on the same line. So you only need one here with both buttons inside.

Reply:
Reset buttons are terrible. If you spend 15 minutes filling out a form and accidentally click it, riperino. Instead I think you should replace it with a cancel link, and give it the class `button-tertiary` rather than `button`. The cancel link could point to the maintenance events index, with `${pageContext.request.contextPath}/maintenance`

Reply:
I had a similar thought. Have you developed the css for button-tertiary yet? Currently in testing its just a link.

Reply:
I might not have added it in yet, but I do have css for a tertiary button somewhere. It is supposed to be very simplistic. Just a basic link but with the same padding as a button, so it looks like it belongs beside one.

Reply:
@JacobNolan1 (just so you get a notification)